lib/src/visitor/recursive_ast.dart

```diff
@@ -24,10 +24,6 @@ abstract class RecursiveAstVisitor extends RecursiveStatementVisitor
     implements ExpressionVisitor<void> {
   const RecursiveAstVisitor();
 
-  void visitAtRootRule(AtRootRule node) {
-    super.visitAtRootRule(node);
```

Comment:
Instead of deleting this, shouldn't this also visit the query if it's non-null?

Reply:
Good catch!